### DISCUSSION TITLE
custom PyTorch network cannot be 'aquant' sucessfully

### DISCUSSION THREAD

**Discussion by noahzn** [bitcraze/discussions#577] [Q&A] [answered]
Hi, I want to quantize my PyTorch model based on the classification demo. I also modified the makefile to set the correct model path.
I have converted the model to .onnx version, and I run ` sudo docker run --rm -v ${PWD}:/module aideck-with-autotiler tools/build/make-example examples/ai/drone clean model build image`. 
1. The error is: 
`    matcher - ++ fusion insert_copies modified graph    
    aquant - input file ['samples3/0copy.jpg']     
    graph_executer - execute uncached: quantization mode none 
    EXCEPTION of type 'error' occurred with message: 'unpack_from requires a buffer of at least 118621708 bytes for unpacking 4 bytes at offset 118621704 (actual buffer size is 822137)'
    To enable full traceback, run the following command: 'set debug true'`

@hmllr  could you help me? Thanks!
**update1:** Since I can quantize the model using a local GAP_SDK (not through the aideck docker), I try to put the generated file `.json` file into the BUILD folder, then it says `No module named 'nntool'`

**Comment by noahzn** [comment, marked as answer]
The latest aideck_docker has an old version of gap_sdk, which cannot handle my onnx model. This problem has been solved by using a newer version of sdk.

**Reply by Jool99** [reply]
May I ask how you achieved this? 

Did you upgrade the gap_sdk inside docker or use a native installation of the SDK?

Background: 
I have an onnx model that I can get through nntool with my native install of the gap_sdk but not with the one in the aideck_docker.
But if I try to do the full build with the native SDK I get the following error: 

gap_sdk/tools/autotiler_v3/Emulation/at_api_pmsis.h:206:32: error: storage size of 'default_flash_conf' isn't known
   struct pi_default_flash_conf default_flash_conf;
